src/test/java/pages/HomePage.java

```diff
@@ -20,6 +20,9 @@
     @FindBy(xpath = "//h5[@itemprop= 'name']/a[@title ='Printed Dress']")
```

Comment:
why do select a predefined item on page, what if it disappears? you should select random element or, get via API smthing or...